replay/models/kl_ucb.py

```diff
@@ -0,0 +1,154 @@
+import math
+
+from typing import Optional
+from replay.models import UCB
+from pyspark.sql.types import DoubleType
+from pyspark.sql.functions import udf
+from scipy.optimize import root_scalar
+
+
+class KL_UCB(UCB):
```

Comment:
Thank you for the thorough documentation. What do you mean by `Single actor`?

Reply:
I would suggest to remove term `actor`. It is more likely to be used in Deep RL (quick googling "multi actor UCB" gives no result), also we don't use term actor in UCB code. 
For me without "actor" it looks clear, like `model treat log as non-personalized, e.g. item scores are the same for all users`.

Reply:
ok, got it